packaging_automation/common_tool_methods.py

```diff
@@ -32,10 +35,33 @@ def release_all_repos():
         repo.close()
 
 
+class PackageType(Enum):
+    deb = 1,
+    rpm = 2
+
+
+BASE_PATH = pathlib2.Path(__file__).parents[1]
+
+
 def get_spec_file_name(project_name: str) -> str:
     return f"{project_name}.spec"
 
 
+def get_minor_project_version(project_version: str) -> str:
+    project_version_details = get_version_details(project_version)
+    return f'{project_version_details["major"]}.{project_version_details["minor"]}'
+
+
+def get_version_number(version: str, fancy: bool, fancy_release_count: int) -> str:
```

Comment:
Will we ever call either of `get_version_number` & `get_version_number_with_project_name` with `fancy = False`.
If not, I think we can remove `fancy: bool` params and rename those two functions as such:

`get_version_number` -> `get_fancy_version`
`get_version_number_with_project_name` -> `get_fancy_version_with_project_name`

Reply:
Personally, I don't quite like making a function no-op depending on the parameter.
So I think we can do something like that for pg-autofailover, what do you think ?

```python
if pg_autofailover:
    version = get_fancy_version(version)
    # or version = get_fancy_version_with_project_name(version )
```

If that makes sense to you too, then I think we can remove `fancy` param and rename those two functions.

Reply:
fancy version number is needed to get fancy version. I got as parameter